config/jobs/kubernetes/sig-cloud-provider/azure/release-1.19.yaml

```diff
@@ -437,9 +438,9 @@ periodics:
     path_alias: k8s.io/kubernetes
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/krte:v20210319-e46e31c-master
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210312-67f589a-master
```

Comment:
I think you need go-canary to get go 1.16

Reply:
https://github.com/kubernetes/test-infra/blob/5209ee96af140395dbc80f46e7f78f2782862051/config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml#L13

Reply:
`gcr.io/k8s-testimages/kubekins-e2e:v20210312-67f589a-master` also has Go 1.16.1. The reason why it didn't work in the past is that we use the kubekins-e2e based on the Kubernetes release, which has a different Go version from master